### DISCUSSION TITLE
How to reuse different SR models

### DISCUSSION THREAD

**Discussion by Morning138** [Monaco12138/sr#3] [Q&A] [answered]
Thank you for your code, i wonder how to reuse different sr model.
Now there are only edsr for example, i want to replace it with rcan model

**Comment by Monaco12138** [comment, marked as answer]
Surely, to reuse different sr model, you should first add the related model.py to ./models and add the register function.
In ./models/models.py, a register function automatically login the related function before you use it.
Then, adjust the ./models/__init__.py to import the file.
After that, you can easily reuse different sr models.

**Reply by Morning138** [reply]
Thank you for your reply, i will try it later.
